README.md

```diff
@@ -318,6 +319,8 @@ The following are only in the resource subject (that is, the base subject).
   literal: <literal>,
   lang: <language for literal or URI label>,
   -> langLabel: <label for language>,
+  validationRegex: <regex for literal validation, as a string with backslashes double-escaped (e.g. "^\\d+$" becomes /^\d+$/)>,
+  validationDataType: <uri for integer, dateTime, etc. for literal validation>,
```

Comment:
I trust this and the new lines 294-297 are sufficient documentation of the new properties.